include/fiction/algorithms/simulation/sidb/is_operational.hpp

```diff
@@ -294,7 +275,7 @@ is_operational(const Lyt& lyt, const std::vector<TT>& spec, const is_operational
     assert(lyt.num_pis() > 0 && "skeleton needs input cells");
     assert(lyt.num_pos() > 0 && "skeleton needs output cells");
 
```

Comment:
When I work with multi-output truth tables, I always add an `assert` to check that they all have the same number of variables. See `one_pass_synthesis.hpp` for instance.